ethereum/executionclient/src/main/java/tech/pegasys/teku/ethereum/executionclient/schema/PayloadAttributesV2.java

```diff
@@ -31,4 +36,34 @@ public PayloadAttributesV2(
     super(timestamp, prevRandao, suggestedFeeRecipient);
     this.withdrawals = withdrawals;
   }
+
+  public static Optional<PayloadAttributesV2> fromInternalPayloadBuildingAttributesV2(
+      Optional<PayloadBuildingAttributes> payloadBuildingAttributes) {
+    return payloadBuildingAttributes.map(
+        (payloadAttributes) ->
+            new PayloadAttributesV2(
+                payloadAttributes.getTimestamp(),
+                payloadAttributes.getPrevRandao(),
+                payloadAttributes.getFeeRecipient(),
+                PayloadAttributesV2.getWithdrawals(payloadAttributes.getWithdrawals())));
+  }
+
+  public static List<WithdrawalV1> getWithdrawals(
+      final Optional<List<Withdrawal>> maybeWithdrawals) {
+    if (maybeWithdrawals.isEmpty()) {
+      return List.of();
```

Comment:
Shouldn't this should be `null` when withdrawals aren't supported not an empty list?

Reply:
mm... not sure...  soemthing to think more on...